server/user_account/tests.py

```diff
@@ -90,11 +125,14 @@ def setUp(self):
 
     def api_authentication_valid_token(self):
         """ Added valid token header to request """
+        # pylint: disable=no-member
```

Comment:
The member 'credentials' is not being detected. I tested it and the code is indeed working and this call is required for the test to work.

Reply:
So are we supposed to be passing credentials somehow? I don't get why pylint is flagging this instance?

Reply:
Its just a method for the inherited class. Probably a bug or something with pylint.

Reply:
[Seems to be a false positive](http://pylint-messages.wikidot.com/messages:e1101)